runtime/doc/options.txt

```diff
@@ -7004,6 +7004,10 @@ A jump table for the options with a short description can be found at |Q_op|.
 	When changing something that is used in 'winbar' that does not trigger
 	it to be updated, use |:redrawstatus|.
 
+	Floating windows do not use the global value of 'winbar'. The
+	window-local value of 'winbar' must be set for a floating window to
+	have a window bar.
```

Comment:
> you can also use `setglobal winbar=%!ActiveWinbar()` instead.

would be worth mentioning this `setglobal` example explicitly in the docs here. Because `setglobal` is rather uncommon, and it's uncommon for options to change their behavior in this way (though I think it's a good change).